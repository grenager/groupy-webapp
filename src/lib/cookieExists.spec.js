import Cookies from 'universal-cookie';
import groupyCookie from './cookie';

const cookieExists = require('./cookieExists');

test('can we instantiate universal cookie?', () => {
  const cookies = new Cookies();
  expect(cookies.HAS_DOCUMENT_COOKIE).toBe(true);
});

test('proper cookie data is correct', () => {
  expect(groupyCookie).toMatchSnapshot();
  expect(groupyCookie.userID).toEqual('5e62ade15014b544a50af943');
});

test('groupyCookie object should have properties token and userID', () => {
  expect(groupyCookie).toHaveProperty('token');
  expect(groupyCookie).toHaveProperty('userID');
})
