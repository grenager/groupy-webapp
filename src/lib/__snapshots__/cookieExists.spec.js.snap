// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`proper cookie data is correct 1`] = `
Object {
  "token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySUQiOiI1ZTYyYWRlMTUwMTRiNTQ0YTUwYWY5NDMiLCJpYXQiOjE1ODg3OTc5MTh9.ikGhUB9-TWq7IidKEg76FVFeuxsKIzGJ10BjexCQL9Q",
  "userID": "5e62ade15014b544a50af943",
}
`;
